requirements.txt

```diff
@@ -5,7 +5,8 @@ boto3
 botocore
 cerberus
 docopt
-requests
+google-api-python-client
+requests==2.14.1
```

Comment:
> python setup.py develop
error: requests 2.18.1 is installed but requests~=2.14.1 is required by {'msrest'}